mopidy_youtube/backend.py

```diff
@@ -49,7 +52,7 @@ def resolve_url(url, stream=False):
     try:
         video = pafy.new(url)
         if not stream:
-            uri = 'youtube:video/%s.%s' % (
+            uri = video_uri_prefix + '/%s.%s' % (
```

Comment:
`'%s/%s.%s' % (video_uri_prefix, safe_url(video.title), video.videoid)` might be better than mixing concatenation and string formatting.